app/views/comments/_comment_header.html.erb

```diff
@@ -0,0 +1,34 @@
+<header class="comment__header">
+  <a href="/<%= comment.user.username %>" class="crayons-link crayons-link--secondary flex items-center fw-medium">
+    <span class="js-comment-username"><%= comment.user.name %></span>
+    <% if commentable_author_is_op?(commentable, comment) %>
+      <span class="crayons-tooltip inline-block" data-tooltip="<%= get_ama_or_op_banner(commentable) %>">
+        <%= inline_svg_tag("small-medal.svg", aria: true, class: "crayons-icon", title: get_ama_or_op_banner(commentable)) %>
+      </span>
+    <% end %>
+  </a>
+
+  <%= render "comments/comment_date", decorated_comment: decorated_comment %>
+
+  <div class="comment__dropdown">
+    <button class="dropbtn crayons-btn crayons-btn--s crayons-btn--ghost-dimmed crayons-btn--icon" aria-label="Toggle dropdown menu">
+      <%= inline_svg_tag("overflow-horizontal.svg", aria: true, class: "crayons-icon pointer-events-none", title: "Dropdown menu") %>
+    </button>
+    <div class="dropdown">
+      <div class="crayons-dropdown p-1 right-1 s:right-0 s:left-auto fs-base">
+        <a href="<%= comment.path %>" class="crayons-link crayons-link--block">Permalink</a>
```

Comment:
This is another case where our `URL` module could help:

```suggestion
        <a href="<%= URL.comment(comment) %>" class="crayons-link crayons-link--block">Permalink</a>
```

Yes, this is slightly longer, but at least all URLs are constructed in consistent ways, so if there's ever a need for change we can do it in a single place.